__tests__/app.test.js

```diff
@@ -184,4 +184,40 @@ describe('/api/articles/:article_id/comments', () => {
         expect(res.body.msg).toBe('INVALID INPUT')
       })
     })
-  })
\ No newline at end of file
+  })
+
+  describe('POST /api/articles/:article_id/comments', () => {
+    test('POST /api/articles/:article_id/comments returns an 201 status code ', () => {
+      return request(app)
+      .post('/api/articles/1/comments')
+      .send({username: 'butter_bridge', body: 'Hello There!'})
+      .expect(201)
```

Comment:
You want to make sure you are checking the comment has all the expected properties 👀